### PR TITLE
Update some option docstrings for accuracy.

### PR DESCRIPTION
**Pull request by aquarapid**
Signed-off-by: Jacques Grove <aquarapid@gmail.com>

### CODE REVIEW

File: go/vt/vttablet/tabletserver/tabletenv/config.go [hunk 1]
```diff
@@ -149,8 +149,8 @@ func init() {
 	flag.BoolVar(&currentConfig.CacheResultFields, "enable-query-plan-field-caching", defaultConfig.CacheResultFields, "This option fetches & caches fields (columns) when storing query plans")
 
 	flag.DurationVar(&healthCheckInterval, "health_check_interval", 20*time.Second, "Interval between health checks")
-	flag.DurationVar(&degradedThreshold, "degraded_threshold", 30*time.Second, "replication lag after which a replica is considered degraded (only used in status UI)")
```
Comment (deepthi) [inline go/vt/vttablet/tabletserver/tabletenv/config.go:152]: Is this really used anywhere other than Status UI?

File: go/vt/discovery/replicationlag.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ var (
 	// lowReplicationLag defines the duration that replication lag is low enough that the VTTablet is considered healthy.
 	lowReplicationLag             = flag.Duration("discovery_low_replication_lag", 30*time.Second, "the replication lag that is considered low enough to be healthy")
 	highReplicationLagMinServing  = flag.Duration("discovery_high_replication_lag_minimum_serving", 2*time.Hour, "the replication lag that is considered too high when selecting the minimum num vttablets for serving")
-	minNumTablets                 = flag.Int("min_number_serving_vttablets", 2, "the minimum number of vttablets that will be continue to be used even with low replication lag")
+	minNumTablets                 = flag.Int("min_number_serving_vttablets", 2, "the minimum number of vttablets for each replicating tablet_type (e.g. replica, rdonly) that will be continue to be used even with replication lag above degraded_threshold")
```
Comment (deepthi) [inline go/vt/discovery/replicationlag.go:30]: Strictly speaking we are checking against the low_replication_lag value and not degraded_threshold.

File: go/vt/vttablet/tabletserver/tabletenv/config.go (hunk 1 above)
Comment (aquarapid) [reply go/vt/vttablet/tabletserver/tabletenv/config.go:152]: We populate the healthStreamer with this info, so potentially, yes?

Comment (deepthi) [reply go/vt/vttablet/tabletserver/tabletenv/config.go:152]: oh yes, that is new. This is fine.

File: go/vt/discovery/replicationlag.go (hunk 2 above)
Comment (aquarapid) [reply go/vt/discovery/replicationlag.go]: Fixed;  and expanded a bit more.

**Pull request merged**